### PR TITLE
Feat suggestion emoji 0904

### PR DESCRIPTION
**Pull request by phong9xNcc**

### CODE REVIEW

File: libs/components/src/lib/components/EmojiPicker/EmojiSuggestion.tsx [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
-	const { emojis, setEmojiSuggestion, setIsEmojiListShowed, isEmojiListShowed, setIsFocusEditorStatus, setTextToSearchEmojiSuggesion } =
-		useEmojiSuggestion();
+	const {
+		emojis,
+		setEmojiSuggestion,
+		setIsEmojiListShowed,
+		isEmojiListShowed,
+		setKeyCodeFromKeyBoardState,
+		setTextToSearchEmojiSuggesion,
+		setKeyboardPressAnyButtonStatus,
```
Comment (nguyentrannhan) [inline libs/components/src/lib/components/EmojiPicker/EmojiSuggestion.tsx:16]: Indent

File: libs/components/src/lib/components/MessageBox/index.tsx [hunk 2]
```diff
@@ -341,48 +341,47 @@ function MessageBox(props: MessageBoxProps): ReactElement {
...
 	// }, [isEmojiListShowed, textToSearchEmojiSuggestion]);
 
-	const clearSuggestionEmojiAfterSendMessage = () => {
-		setIsEmojiListShowed(false);
-		setEmojiSuggestion('');
-		setTextToSearchEmojiSuggesion('');
-		setIsFocusEditorStatus(false);
-		setEditorState(() => EditorState.createEmpty());
-	};
+	// const clearSuggestionEmojiAfterSendMessage = () => {
```
Comment (nguyentrannhan) [inline libs/components/src/lib/components/MessageBox/index.tsx]: Delete this

File: libs/components/src/lib/components/MessageBox/index.tsx [hunk 3]
```diff
@@ -341,48 +341,47 @@ function MessageBox(props: MessageBoxProps): ReactElement {
...
-	function findSyntaxEmoji(contentText: string): string | null {
-		const regexEmoji = /:[^\s]+(?=$|[\p{Emoji}])/gu;
-		const emojiArray = Array.from(contentText.matchAll(regexEmoji), (match) => match[0]);
-		if (emojiArray.length > 0) {
-			return emojiArray[0];
-		}
-		return null;
-	}
+	// function clickEmojiSuggestion() {
+	// 	if (!emojiPicked) {
```
Comment (nguyentrannhan) [inline libs/components/src/lib/components/MessageBox/index.tsx]: Delete this

File: libs/components/src/lib/components/EmojiPicker/EmojiSuggestion.tsx (hunk 1 above)
Comment (phong9xNcc) [reply libs/components/src/lib/components/EmojiPicker/EmojiSuggestion.tsx:16]: done

**Pull request merged**